src/OpenCqrs.Bus.ServiceBus/Factories/MessageFactory.cs

```diff
@@ -1,20 +1,47 @@
-﻿using System.Text;
+﻿using System.Collections.Generic;
+using System.Text;
 using Microsoft.Azure.ServiceBus;
 using Newtonsoft.Json;
 
 namespace OpenCqrs.Bus.ServiceBus.Factories
 {
     public class MessageFactory : IMessageFactory
     {
+        public static readonly string AssemblyQualifiedNamePropertyName = "AssemblyQualifiedName";
         /// <inheritdoc />
         public Message CreateMessage<TMessage>(TMessage message) where TMessage : IBusMessage
         {
             var json = JsonConvert.SerializeObject(message);
-            var serviceBusMessage = new Message(Encoding.UTF8.GetBytes(json));
+            var serviceBusMessage = new Message(Encoding.UTF8.GetBytes(json))
+            {
+                ContentType = "application/json"
+            };
 
             if (message.ScheduledEnqueueTimeUtc.HasValue)
```

Comment:
I would avoid the risk of overriding the value later on and I would just check if it has value and is not already in the properties dictionary, in that case I would add it to the dictionary.

Reply:
Make sense.